public/javascripts/notebook/observable.js

```diff
@@ -111,25 +111,27 @@ return new function () {
 
   };
 
-
-
   this.makeObservableHelper = function (id, kind, initialValue) {
-    if(!this.isInitialized()) {
-      // in case the Observable isn't initialized, we differ the creation of the observable
-      var args = Array.prototype.slice.call(arguments);
-      var me = this;
-      console.warn("Delaying the creation of Observable (" + args.join(",") + ") since Observable isn't initialized yet");
-      console.log("Observable not initialized:", me);
-      events.on('Observable.ready', function() {
-        me.makeObservableHelper.apply(me, args);
-      });
-      return;
-    }
     var observable = this.observables[id];
     if (typeof observable === 'undefined') {
     	console.log("Creating new observable (client request): " + id)
       observable = ko[kind](initialValue);
-      this.register_observable(id, observable);
+
+      if(!this.isInitialized()) {
+        // in case the Observable isn't initialized, we differ the creation of the observable
```

Comment:
I'd rephrase this comment: delay registering of knockout observable until `Observable.ready`